tests/Database/DatabaseEloquentBuilderTest.php

```diff
@@ -724,6 +769,26 @@ protected function getMockQueryBuilder()
 
         return $query;
     }
+
+    /**
+     * Get a database connection instance.
+     *
+     * @return Illuminate\Database\Connection
```

Comment:
The file is not in a namespace. Therefor it should not be needed? Or am I wrong?